src/randoop/NoExceptionCheck.java

```diff
@@ -3,8 +3,10 @@
 import randoop.sequence.Execution;
 
 /**
- * Checks that no exception is thrown after a given statement
- * in a sequence.
+ * Checks that an exception is not thrown by a given statement in a sequence. 
+ * This is meant for an error-revealing check where, in fact, an exception
+ * was observed at the statement, and this check is meant to represent that
```

Comment:
"was observed" is at test generation time, but this check will occur when the user runs the generated tests, right?  Clarifying what happens at different points in time would be helpful.